<launch>

	<node name="vrep_combiner" pkg="peg_and_ring" type="combiner" args="" respawn="false" output="screen">>

		<param name="old_plugin" type="bool" value="true"/>
		
		<param name="camera_color_name" type="string" value="/camera/color/image_rect_color"/>
		<param name="camera_depth_name" type="string" value="/vrep/depth"/>
		<param name="pcd_topic" type="string" value="/camera/depth/image"/>
	</node>

</launch>